scripts/ci/update_l10n.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# This script is used to update the l10n files in the repository.
+# It is meant to be run by the CI system.
+
+set -e
+
+
+# Clone the station localization repository if it does not exist yet
+if [ ! -d "station.localization" ]; then
+  git clone git@github.com:ruuvi/station.localization.git
+fi
+
+json_file="./station.localization/station.localization.json"
+
+append_xml() {
+  local filename=$1
+  local ident_android=$2
+  local source_string=$3
+
+  # Replace newline characters \\n with \\\\n
+  result=${source_string//\\n/\\\\n}
+
+  # Replace single quotes with escaped single quotes
+  result=${source_string//\'/\\\'}
+
+  # Replace Unicode ampersand with escaped Unicode ampersand
+  result=${result//&/\\u0026}
+
+  # TODO: @rinat-enikeev does not know for what is it. @denisandreev please check
+  # https://github.com/ruuvi/station.localization/blob/master/localize.converter.android/src/model/TranslationString.kt#L35-L41
```

Comment:
probably related to https://github.com/ruuvi/com.ruuvi.station/pull/1155#discussion_r1435847683